composer.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name" : "mouf/composer-validator",
+	"description" : "This package contains a composer validator for mouf interface.",
+	"type" : "mouf-library",
+	"authors" : [{
+			"name" : "Xavier HUBERTY",
+			"email" : "x.huberty@thecodingmachine.com",
+			"homepage" : "http://mouf-php.com"
+		}
+	],
+	"keywords" : [
+		"mouf",
+		"composer",
+		"validator"
+	],
+	"homepage" : "https://github.com/thecodingmachine/composer-validator",
+	"license" : [
+		"MIT"
+	],
+	"require" : {
+		"php" : ">=5.3.0",
+		"composer/composer": "*"
+	},
+	"autoload" : {
+		"psr-4" : {
+			"Mouf\\Composer\\Validator\\" : "src/"
+		}
+	},
+   "minimum-stability": "dev",
+    "prefer-stable": true,
+	"extra" : {
+		"mouf" : {
+			"logo" : "logo.png"
+		}
+	}
+}
```

Comment:
Also, there is no need to commit composer.lock for a library (unless you face integration problems with Scrutinizer)